_includes/content/non-helm-manifests/install-calico.ps1

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2020 Tigera, Inc. All rights reserved.
```

Comment:
I don't think this belongs in the non-helm-manifests section. Mostly because it isn't a Kubernetes manifest :) 

but also because it doesn't seem to use any templating? 

Why not just put the entire contents of this file straight into `manifests/windows/install-calico.ps1` and remove the indirection? I think that would be nicer, unless I'm missing something!

Reply:
I moved it out of `_includes/content/non-helm-manifests` and into `_includes/non-helm-manifests` -- does that look right?

There is one line where we template in the current version for the zip file name.

Reply:
Ah, I missed that we are in fact templating this. 

I still wonder if it should be in "manifests" since it's not a manifest. Is there corresponding doc for this script? How are we going to be instructing users to use this?

Reply:
> I still wonder if it should be in "manifests" since it's not a manifest. 

Good point! I've stuck it `scripts/`.

> Is there corresponding doc for this script? How are we going to be instructing users to use this?

There's an in-progress PR here https://github.com/projectcalico/calico/pull/3851